easybuild/easyconfigs/x/XZ/XZ-5.2.3-gimkl-2017a.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'XZ'
+version = '5.2.3'
+
+homepage = 'http://tukaani.org/xz/'
+description = "xz: XZ utilities"
+
+toolchain = {'version': '2017a', 'name': 'gimkl'}
+
+sources = [SOURCELOWER_TAR_BZ2]
+source_urls = ['http://tukaani.org/xz/']
```

Comment:
@valtandor Please use the patch for XZ to avoid breaking the RPM command, see #4179 

```python
patches = ['XZ-5.2.2_compat-libs.patch']